README.md

```diff
@@ -38,3 +38,5 @@ This project is licensed under [Apache License, Version 2.0](LICENSE).
 ## Reporting a Vulnerability
 
 Vulnerabilities can be reported by sending an email to security@spiffe.io. A confirmation email will be sent to acknowledge the report within 72 hours. A second acknowledgement will be sent within 7 days when the vulnerability has been positively or negatively confirmed.
+
+## Parameters
```

Comment:
This can be removed. The root README is not a Helm Chart README, but a repository README. It also won't end up in artifacthub

Reply:
Doh! :sweat_smile: Done.